modules/merqury/meta.yml

```
@@ -0,0 +1,101 @@
+name: "merqury"
+description: k-mer based assembly evaluation.
+keywords:
+  - "k-mer"
+  - "assembly"
+  - "evaluation"
+tools:
+  - "merqury":
+      description: "Evaluate genome assemblies with k-mers and more."
+      homepage: "None"
+      documentation: "None"
+      tool_dev_url: "https://github.com/marbl/merqury"
+      doi: ""
```

```suggestion
      doi: "10.1186/s13059-020-02134-9"
```
as in e.g. https://github.com/nf-core/modules/blob/d8bef6057bc2cf657d33ed496426be8416edd780/modules/antismash/antismashlite/meta.yml#L25?

---

I always forget the metadata. Thanks.